jit/v2/JIT_v2.0.0.yaml

```diff
@@ -411,6 +922,102 @@ paths:
           headers:
             API-Version:
               $ref: '#/components/headers/API-Version'
+        '400':
+          description: |
+            In case decling a `Port Call Service` does not schema validate a `400` (Bad Request) is returned
```

Comment:
In case declining a 'Port Call Service' fails schema validation a 400 (Bad Request) is returned